data/component-list.mezzio.json

```diff
@@ -110,5 +110,12 @@
     "url": "https://docs.mezzio.dev/mezzio-problem-details/",
     "group": "API",
     "description": "PSR-7 Problem Details for HTTP API responses and middleware."
+  },
+  {
+    "name": "CORS",
+    "package": "mezzio/mezzio-cors",
+    "url": "https://docs.mezzio.dev/mezzio-cors/",
+    "group": "",
```

Comment:
Let's use either "API" or "Authorization" here; it could fit with either one. Generally speaking, you'll only use them for API calls from a browser, and they're to determine whether the incoming request is allowed (hence, authorized). Take your pick.

Reply:
Was thinking about `Authorization` aswell. Gonna take that.